src/locale/en.json

```diff
@@ -319,7 +319,8 @@
 		"po-special-characters": "A purchase order should not contain dots (.), colons ( : ), or hashes ( # )",
 		"cancelBooking": "There was an error cancelling your booking request. Please try again.",
 		"registrationNotFound": "You attempted to cancel a booking you were not registered.",
-		"empty-payment-method": "Please enter your payment details."
+		"empty-payment-method": "Please enter your payment details.",
+		"otherareaofworksmultiselection": "You can not select an other area of work as well as 'I don’t know' option"
```

Comment:
Could you please rename the key "otherareaofworksmultiselection" to "otherareaofworksidontknowmultiselection".